### PR TITLE
Use go mod

### PR DESCRIPTION
**Pull request by RamLavi**
migrating to go modules dependency management tool

### CODE REVIEW

File: tool.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// +build tools
```
Comment (phoracek) [inline tool.go]: Make sure to run gofmt on this one. Also, please add a newline at the end.

File: hack/vendor-tools.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+export GOFLAGS=-mod=vendor
```
Comment (phoracek) [inline hack/vendor-tools.sh]: These two are already exported by the Makefile, you can drop them.

**Comment by RamLavi** [comment]
@phoracek, why did you merge if I didn't fix the comments yet ? was is by mistake?

**Comment by phoracek** [comment]
Yeah, I was merging from my phone and forgot about those. It's just a nit, we can address it later.

**Pull request merged**